src/TConsole.cpp

```diff
@@ -1546,9 +1546,8 @@ int TConsole::select(const QString& text, int numOfMatch)
     }
 
     if (mudlet::debugMode) {
-        TDebug(QColor(Qt::darkMagenta), QColor(Qt::black)) << "\nline under current user cursor: " >> 0;
```

Comment:
Can't retain the leading `\n` as that won't sit well (IMHO) with something prefixed onto the line.